js/rooms.js

```diff
@@ -45,7 +45,33 @@ $(document).ready(function() {
 				success: function(data) {
 					$('#app-navigation ul').html('');
 					data.forEach(function(element) {
-						$('#app-navigation ul').append('<li><a href="#'+escapeHTML(element['id'])+'">'+escapeHTML(element['name'])+' <span class="utils">' + escapeHTML(element['count']) + '</span></a></li>');
+						$('#spreedme-room-list').append(
+							'<li id="'+escapeHTML(element['name'])+'">'+
```

Comment:
this code should be applied to here https://github.com/nextcloud/spreed/blob/e76242bee99b2a710e0b95e5e70ef59abf70553f/js/views/roomlistview.js#L29